nss-cohort-36/c36-class-website
# Sp update student json

## Type of change
New feature (non-breaking change which adds functionality to see pics)

## Testing Instructions
Make sure to commit local changes in your branch before switching to master.

```
git checkout master
git fetch --all
git checkout sp-update-student-json
Check to see that site loads and all student cards are there
```

## Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
